tools/email.py

```diff
@@ -54,8 +54,24 @@ def end_email(run_name, samples):
     send_email(subject, body)
 
 
-#def error_email():
+def error_email(run_name, ok_samples, not_ok_samples):
+    """Send an email about which samples have failed and which samples have succeeded"""
 
+    subject = f'WGS Somatic end mail {run_name}'
 
+    body = '\n'.join([f'WGS somatic has finished successfully for the following samples in run {run_name}:',
+                      '',
+                      f'{new_line}{new_line.join(ok_samples)}',
+                      '',
+                      'The following samples have not finished correctly:',
+                      '',
+                      f'{new_line}{new_line.join(not_ok_samples)}',
+                      '',
+                      'Errors concerning these samples will be investigated.',
+                      '',
+                      'Best regards,',
+                      'CGG Cancer'])
```

Comment:
I'd use triple quotes text block:
    """
    text here
    """

Reply:
how do I handle the f strings if I make a text block instead?

Reply:
solved in commit @ 785922c